### PR TITLE
DOCS-91 - Playbook payloads

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds a new section on playbook payloads to this article:
https://help.sumologic.com/docs/platform-services/automation-service/automation-service-playbooks/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-91](https://sumologic.atlassian.net/browse/DOCS-91)

### CODE REVIEW

File: sidebars.ts [hunk 1]
```diff
@@ -1090,7 +1090,6 @@ module.exports = {
             'alerts/monitors/muting-schedules',
             'manage/ingestion-volume/monitor-ingestion-receive-alerts',
             'alerts/monitors/use-playbooks-with-monitors',
-            'alerts/monitors/automation-payload-variables'
```
Comment (kimsauce) [inline sidebars.ts:1093]: should docs/platform-services/automation-service/automation-service-playbooks.md be added to` platform-services `sidebars?

**Review by kimsauce — APPROVED** [review]
did [one fix](https://github.com/SumoLogic/sumologic-documentation/pull/4650/commits/1d5e55a04ce619aaf4701a64e4c3c5dca288284d)

File: sidebars.ts (hunk 1 above)
Comment (jpipkin1) [reply sidebars.ts:1093]: @kimsauce, that file is already in the platform services sidebar:
https://help.sumologic.com/docs/platform-services/automation-service/automation-service-playbooks/

**Comment by jpipkin1** [comment]
Thanks for the fix, @kimsauce.

**Pull request merged**